### PR TITLE
test(chips): Add Jasmine foundation test to chip

### PR DESCRIPTION
**Pull request by allan-chen**
Project import generated by Copybara.

### CODE REVIEW

File: packages/mdc-chips/chip/test/foundation.test.ts [hunk 1]
```diff
@@ -0,0 +1,703 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+
+import {verifyDefaultAdapter} from '../../../../testing/helpers/foundation';
+import {setUpFoundationTest, setUpMdcTestEnvironment} from '../../../../testing/helpers/setup';
```
Comment (abhiomkar) [inline packages/mdc-chips/chip/test/foundation.test.ts]: Does Copybara transform work for 4 level depth relative path?

Comment (allan-chen) [reply packages/mdc-chips/chip/test/foundation.test.ts]: It doesn't work - it tries to change `testing` to `mdc-testing`

Comment (abhiomkar) [reply packages/mdc-chips/chip/test/foundation.test.ts]: Did you mean it does work? I assume it worked when you exported the CL :)

Comment (allan-chen) [reply packages/mdc-chips/chip/test/foundation.test.ts]: It doesn't - I had to manually copy and paste and change the import paths for now

Comment (allan-chen) [reply packages/mdc-chips/chip/test/foundation.test.ts]: Export also doesn't work yet for all the other migrated components (i.e. checkbox), I can run a test PR to demonstrate

Comment (allan-chen) [reply packages/mdc-chips/chip/test/foundation.test.ts]: See here: https://github.com/material-components/material-components-web/pull/5376/files

Comment (abhiomkar) [reply packages/mdc-chips/chip/test/foundation.test.ts]: Ah ok. Seems like Copybara config file needs to be updated to transform imports in tests. Exporting to PR should just work without any manually changes.

I'll update the Copybara transformation.

Comment (allan-chen) [reply packages/mdc-chips/chip/test/foundation.test.ts]: Thanks! Sorry about the misleading description

**Pull request merged**